vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -620,6 +634,20 @@ void BatGetMedia::onFetchPublisherFromDBResponse(ledger::Result result,
                                         _2,
                                         _3));
   } else {
+    if (providerType == TWITCH_MEDIA_TYPE) {
+      if (info->name != visit_data.name) {
```

Comment:
We need to check here if `info` is null as  it could be null pointer

Reply:
Done